examples/layer-browser/src/data-samples.js

```diff
@@ -136,12 +136,24 @@ export function getPoints100K() {
   return _points100K;
 }
 
+let _points500K = null;
```

Comment:
are these _points* used somewhere else? (maybe I missed it in other PRs).
if not just return pointGrid(5 * 1e5, [-122.6, 37.6, -122.2, 37.9]); directly?

Reply:
we are just caching them, since it takes time to generate that many points, we are doing same for other points in this file.